src/views/evaluation/Evaluation.js

```diff
@@ -1,15 +1,15 @@
-const NAV = function (option = {}){
+const NAV = function (option = {}, logo = {}){
```

Comment:
Sounds like you could very well have `options.items` and `options.logo` instead. That way, you're keeping the single parameter function header consistent with the other rendering functions, but you're using that single parameter to encapsulate all the information specific to construction of the navigation component. Makes sense?